visual-js/visual/src/utils.ts

```diff
@@ -122,3 +126,101 @@ export const parseRegionsForAPI = async <T>(
     ignoreElements: flattened.filter(isElementIn),
   };
 };
+
+/**
+ * Dom capturing script path
+ */
+
+export const domScriptPath = `${os.tmpdir()}/sauce-visual-dom-capture.js`;
+
+/**
+ * Fetch and save to tmp file dom capturing script
+ */
+
+export const downloadDomScript = async (api: ReturnType<typeof getApi>) => {
```

Comment:
I guess there is noway we can keep that script in memory only ? (to be 100% sure there is no left-overs)

Reply:
_Maybe_ -- but this is designed to do it once for all of the workers during global setup / cleanup during teardown. Otherwise it would download a separate file for each process / worker.